pkg/deploy/controller/deploymentconfig/controller.go

```diff
@@ -396,6 +404,8 @@ func (c *DeploymentConfigController) handleErr(err error, key interface{}) {
 		c.queue.Forget(key)
 		return
 	}
+	// Log stack trace for every error
+	utilerrors.WithStacktrace(err)
```

Comment:
@kargakis this should show the full stack trace for the original error so we can hunt down the conflicts... otherwise all errors in log are pointing to this function which is not very helpful.